bip-0341.mediawiki

```diff
@@ -294,7 +294,43 @@ Examples of preimage for sighashing for each of the sighash modes.
 
 == Deployment ==
 
-TODO
+This BIP is deployed concurrently with [[bip-0342.mediawiki|BIP342]].
```

Comment:
Procedurally, should mention bip340 ?

Reply:
bip340 on itself can not be deployed because it doesn't describe consensus rules. It is deployed only as part of bip341 and 342.

Reply:
Thanks, make sense